spec/routing/posts_routing_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "rails_helper"
+
+describe PostsController do
+  describe 'routing' do
+
+    it 'routes to #index' do
+      expect(get: "/posts").to route_to("posts#index")
+    end
+
+    it 'routes to #show' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.